File: Cpp/1924.cpp
#include "stdio.h"

int main() {
    int month, day, idx;
    int period = 0;

    scanf("%d %d", &month, &day);

    period = (month-1)*30;
    for(int i = 1; i < month; i+=2) period++;
    if(month > 6) period++;
    if(month == 12) period--;
    if(month > 2) period -= 2;
    period += day;

    char *day_of_week[7] = {"SUN", "MON", "TUE", "WED", "THU", "FRI", "SAT"};
    idx = period%7;
    printf("%s", day_of_week[idx]);
    return 0;
}

//오답입니다.

File: Python/1924.py
month, day = input().split()
month = int(month)
day = int(day)

period = 0

if month > 1:
    period += 31
if month > 2:
    period += 28
if month > 3:
    period += 31
if month > 4:
    period += 30
if month > 5:
    period += 31
if month > 6:
    period += 30
if month > 7:
    period += 31
if month > 8:
    period += 31
if month > 9:
    period += 30
if month > 10:
    period += 31
if month > 11:
    period += 30
period += day

idx = period % 7

ans = ["SUN", "MON", "TUE", "WED", "THU", "FRI", "SAT"]
print(ans[idx])